e2e-pw/src/oss/poms/modal/looker-3d-controls.ts

```diff
@@ -0,0 +1,35 @@
+import { Locator, Page, expect } from "src/oss/fixtures";
+import { ModalPom } from ".";
+
+export class Modal3dControlsPom {
+  readonly page: Page;
+  readonly assert: Modal3dControlsAsserter;
+  readonly controls: Locator;
+  readonly optionsPanel: Locator;
+  readonly time: Locator;
+  readonly playPauseButton: Locator;
```

Comment:
Something _looks_ wrong here... modal3d <---> play/pause + time?

Reply:
Not to mention that this super sus POM isn't used anywhere...

Reply:
I didn't mean to commit this. This was scratch code 😅 removing